src/CoreShop/Bundle/SEOBundle/DependencyInjection/Compiler/ExtractorRegistryServicePass.php

```diff
@@ -40,9 +40,13 @@ public function process(ContainerBuilder $container)
                 $attributes[0]['type'] = Container::underscore(substr(strrchr($definition->getClass(), '\\'), 1));
             }
 
+            if (!isset($attributes[0]['priority'])) {
+                $attributes[0]['priority'] = 1000;
```

Comment:
Why 1000? If I'm going to add my new extractor, i need to define at least 1001 to overrule system extractors. Feels quite odd. Maybe just `0`?

Reply:
The higher the number the lower the priority. So, with 999 your extractor would overrule